Remove python from LIBS, since the CrOS build doesn't need it.
Bug: https://crbug.com/1287995

--- hplip-3.21.8/configure.in
+++ hplip-3.21.8/configure.in
@@ -659,9 +659,6 @@
 SAVE_CFLAGS=$CFLAGS
 SAVE_LIBS=$LIBS
 
-CFLAGS="$CFLAGS $PYTHON_INCLUDES"
-LIBS="$LIBS $PYTHON_LIBS"
-
 AC_TRY_LINK(,[ ], AC_MSG_RESULT(yes); python_includes="ok", AC_MSG_ERROR([no: PYTHON DISABLED], 6))
 
 if test "X$python_includes" != "Xok" ; then
